engine-nextgen/src/main/kotlin/graphql/nadel/enginekt/blueprint/NadelFieldInstruction.kt

```diff
@@ -104,7 +97,6 @@ data class NadelBatchHydrationFieldInstruction(
     override val hydratedFieldDef: GraphQLFieldDefinition,
     override val actorService: Service,
     override val queryPathToActorField: NadelQueryPath,
-    override val actorFieldDef: GraphQLFieldDefinition,
```

Comment:
Can we rename `overallActorFieldDef` → `actorFieldDef`?

I think it might be a little confusing, but longer term it'll be redundant to explicitly state `overall`.

Reply:
✅